src/sympc/approximations/sigmoid.py

```diff
@@ -19,6 +19,9 @@ def sigmoid(tensor: MPCTensor, method: str = "exp") -> "MPCTensor":
 
     Returns:
         tensor: the calulated sigmoid value
+
+    Raises:
+        ValueError: if the given method is not supported
```

Comment:
you can put the reference of the paper in the docsting as well :)

Reply:
Done!